reference/orders.v2.oas2.yml

```diff
@@ -4129,6 +4136,10 @@ components:
           description: Indicates whether the order is deleted/archived. When set to true in a PUT request, it has the same result as the DELETE an order request.
           example: false
           type: boolean
+        is_tax_inclusive_pricing:
+          description: Indicates whether the order total is inclusive of tax.
```

Comment:
Probably this description would be more meaningful: 

```
Indicate whether the order's base prices are inclusive of tax. 

If true, the base prices are inclusive of tax, and the values of `subtotal_inc_tax`, `shipping_cost_inc_tax`, `handling_cost_inc_tax`, `wrapping_cost_inc_tax` and `total_inc_tax` are not estimated but actual values and can be reliable for accounting purposes.

If false, the base prices are exclusive of tax, and the values of `subtotal_ex_tax`, `shipping_cost_ex_tax`, `handling_cost_ex_tax`, `wrapping_cost_ex_tax` and `total_ex_tax` are not estimated but actual values and can be reliable for accounting purposes.
```

@WillemHoman @bc-vincent-zhao @BC-SEven WDYT about this description, any more info we can add ?

Reply:
We could also update the descriptions of those other properties directly to reference back to this one